src/clarity-angular/dropdown/dropdown-item.ts

```diff
@@ -14,17 +14,23 @@ import { Dropdown } from "./dropdown";
 @Directive({
     selector: "[clrDropdownItem]",
     host: {
-        "[class.dropdown-item]" : "true"
+        "[class.dropdown-item]" : "true",
+        "[class.expandable]" : "expandable",
+        "[class.active]" : "active"
     }
 })
 export class DropdownItem {
 
+    expandable: boolean = false;
+    active: boolean = false;
+
     constructor(private _dropdown: Dropdown, private el: ElementRef) {
     }
 
     @HostListener("click")
     onDropdownItemClick(): void {
-        if (this._dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")) {
+        if (this._dropdown.isMenuClosable && !this.el.nativeElement.classList.contains("disabled")
+                && !this.el.nativeElement.classList.contains("expandable")) {
```

Comment:
Note for later: we really need to clean this up. Checking for `.disabled` on the native classlist is really bad news. 😕 

That said, in your case, you have absolutely no excuse to use this pattern. You literally have an `expandable` property already on your component. So please use that instead.